tests/cupy_tests/padding_tests/test_pad.py

```diff
@@ -21,6 +22,11 @@ class TestPadDefault(unittest.TestCase):
     def test_pad_default(self, xp, dtype):
         array = xp.array(self.array, dtype=dtype)
 
+        if (xp.dtype(dtype).kind in ['i', 'u'] and
+                self.mode in ['mean', 'linear_ramp']):
+            # TODO: can remove this skip once cupy/cupy/#2330 is merged
+            return array
```

Comment:
How about calling `self.skipTest`?

Reply:
Sorry, the function decorated with `@testing.numpy_cupy_array_equal` can't use self.skipTest`. 🙇

Reply:
Hi @asi1024, I have reverted the skipTest change. Can you run the CI again? Thanks